src/Blocks/BlockBigFlower.h

```diff
@@ -54,7 +59,11 @@ class cBlockBigFlowerHandler :
 		NIBBLETYPE FlowerMeta = Meta & 0x7;
 		if (!a_Player->IsGameModeCreative())
 		{
-			if (((FlowerMeta == 2) || (FlowerMeta == 3)) && (a_Player->GetEquippedItem().m_ItemType == E_ITEM_SHEARS))
+			if (
+				((FlowerMeta == E_META_BIG_FLOWER_DOUBLE_TALL_GRASS) ||
+				(FlowerMeta == E_META_BIG_FLOWER_LARGE_FERN)) &&
+				(a_Player->GetEquippedItem().m_ItemType == E_ITEM_SHEARS)
```

Comment:
This is poorly split, it hides the different levels of the condition operators. Consider instead:
```cpp
if (
	(a_Player->GetEquippedItem().m_ItemType == E_ITEM_SHEARS) &&
	(
			(FlowerMeta == E_META_BIG_FLOWER_DOUBLE_TALL_GRASS) ||
			(FlowerMeta == E_META_BIG_FLOWER_LARGE_FERN)
	)
)
```